### DISCUSSION TITLE
How does one troubleshoot forking-renovate?

### DISCUSSION THREAD

**Discussion by pichlermarc** [renovatebot/renovate#21248] [Request Help]
### I'm using the Mend Renovate hosted app on github.com.

- [X] I'm using the Mend Renovate hosted app.

### Was this something which used to work for you, and then stopped?

It used to work, and then stopped

### Describe your problem.

Hi,

we're running forking-renovate on https://github.com/open-telemetry/opentelemetry-js-contrib, but it stopped creating/rebasing PRs around March 14. The last change we made to the config was on March 8. I tried verifying the config locally and it did not report any issues with our config. We tried accessing the logs using the comment on the PR, but it looks like this is [not supported with forking-renovate.](https://github.com/renovatebot/renovate/discussions/18618)

Is there any alternative way we can troubleshoot the issue? :thinking:  

### Relevant debug logs.

<details><summary>Logs</summary>

```
n/a
```

</details>

**Comment by rarkins** [comment]
Try logging into developer.mend.io and see if any information or logs show up there

**Reply by pichlermarc** [reply]
Hi, thanks for the reply :slightly_smiling_face: 
Unfortunately, I end up with a `500 Internal Server Error` when I select `Forking Renovate` and then click on my organization, so no luck there as well.

**Reply by rarkins** [reply]
Could you open developer tools and capture that sequence? I just logged into Forking Renovate from an alternative testing account I used and also get access. I wonder if your account is forbidden from using OAuth or something strange like that?

**Reply by pichlermarc** [reply]
> I wonder if your account is forbidden from using OAuth or something strange like that?

Ah, I think this may be the case; we most likely have some OAuth restrictions. I'll contact some people within the org to see if I can get more info.

**Comment by rarkins** [comment]
Hi, I've been able to access your logs directly and noticed that there's an external host error caused by a cache problem. It looks like a backend problem so we are investigating it. Sorry for the inconvenience

**Reply by rarkins** [reply]
Turns out that was just ephemeral and an unfortunate coincidence. The next job completed successfully. Here's the most recent log: https://gist.github.com/rarkins/741a0e183fba6a5c18a33823aae3c507

I noticed that you only allow PRs to be created in a 3 hour window on Fridays - I'd recommend making that wider, e.g. 5 or 6 hours

**Reply by pichlermarc** [reply]
Looks like this is the log for `open-telemetry/opentelemetry-js`, which should be working fine (`open-telemetry/opentelemetry-js-contrib` is the one having the issues :slightly_smiling_face: ) - I created my own org there I used normal renovate bot to troubleshoot (so I can have access to the logs), and it looks like a docker image that's gone missing from docker hub is what's making the bot fail.

Here are the logs that I used:
https://gist.github.com/pichlermarc/8bbe2cc4d04a0e78df199ce6be0903e5
 
Is it intentional behavior that the bot does not continue when it receives `401` when listing tags for a image using the docker hub api? :thinking:

**Reply by rarkins** [reply]
It actually is intentional. Docker Hub used to have a lot of intermittent errors so we introduced logic that we'd abort rather than keeping going (which could result in autoclosed PRs). When you onboard a repo it will list if there's any such errors and users can choose to fix them or ignore the deps. If in your case the image disappeared later, it's a bit of an unfortunate edge case.

If Docker Hub doesn't send 401/403/404 errors _incorrectly_ anymore then we can perhaps remove the logic to abort a run.

**Reply by pichlermarc** [reply]
Ah, interesting - unfortunate that they send those incorrectly. :confused: 
Looks like replacing the offending image will solve our problem. Thank you for helping us out with this. :slightly_smiling_face:
